docs/linking_to_external_code.md

```diff
@@ -76,6 +76,23 @@ import { assertEquals, runTests, test } from "./deps.ts";
 This design circumvents a plethora of complexity spawned by package management
 software, centralized code repositories, and superfluous file formats.
 
+#### Best practice: Selectively break up libraries.
+
```

Comment:
s/libraries/modules

Reply:
I'm trying to differentiate between an executable project and an importable library. Are these two really interchangeable?